lib/husky_musher/utils/redcap.py

```diff
@@ -48,6 +49,19 @@
 COMPLETE = '2'
 
 
+METRIC_REGISTRY = CollectorRegistry()
+METRIC_REDCAP_REQUEST_SECONDS = Summary(
+    "redcap_request_seconds",
+    "Time spent making requests to REDCap",
+    labelnames = ["function"],
+    registry = METRIC_REGISTRY,
+)
+
+def metric_redcap_request_seconds(function):
+    return METRIC_REDCAP_REQUEST_SECONDS.labels(function.__name__).time()(function)
+
+
+@metric_redcap_request_seconds
```

Comment:
This is pretty smooth 🎉

Reply:
Yeah, the Prometheus instrumentation libraries are fairly nice!